<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Player</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>name_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="name_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Executable:</string>
       </property>
       <property name="buddy">
        <cstring>binary_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="FileChooser" name="binary_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Parameters:</string>
       </property>
       <property name="buddy">
        <cstring>parms_edit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="parms_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="streamingsites_check">
     <property name="text">
      <string>This player supports &amp;video sites</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="onlinetv_check">
     <property name="text">
      <string>This player can play &amp;online TV</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="resolution_label">
       <property name="text">
        <string>Preferred &amp;quality:</string>
       </property>
       <property name="buddy">
        <cstring>resolution_combo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="resolution_combo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Add to &amp;menu:</string>
       </property>
       <property name="buddy">
        <cstring>media_combo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="media_combo"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>158</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QLineEdit</extends>
   <header>filechooser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name_edit</tabstop>
  <tabstop>binary_edit</tabstop>
  <tabstop>parms_edit</tabstop>
  <tabstop>streamingsites_check</tabstop>
  <tabstop>onlinetv_check</tabstop>
  <tabstop>resolution_combo</tabstop>
  <tabstop>media_combo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlayerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlayerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
